src/WorldStorage/WSSAnvil.cpp

```diff
@@ -3167,7 +3183,10 @@ bool cWSSAnvil::LoadEntityBaseFromNBT(cEntity & a_Entity, const cParsedNBT & a_N
 bool cWSSAnvil::LoadMonsterBaseFromNBT(cMonster & a_Monster, const cParsedNBT & a_NBT, int a_TagIdx)
 {
 	float DropChance[5];
-	if (!LoadFloatsListFromNBT(DropChance, 5, a_NBT, a_NBT.FindChildByName(a_TagIdx, "DropChances")))
+	if (
+		!LoadFloatsListFromNBT(DropChance, 5, a_NBT, a_NBT.FindChildByName(a_TagIdx, "DropChances")) &&
+		!LoadFloatsListFromNBT(DropChance, 2, a_NBT, a_NBT.FindChildByName(a_TagIdx, "HandDropChances"))
```

Comment:
I think it would be best if you separated the two codepaths completely:
```cpp
if (LoadFloatsListFromNBT(..., "DropChances")
{
  // old code
}
if (LoadFloatsListFromNBT(..., "HandDropChances")
{
  // Handle hands somehow - perhaps a sum?
}
if (LoadFloatsListFromNBT(..., "ArmorDropChances")
{
  // Set armor drop chances
}
```
(Note that no path returns false now, we've made all the tags optional)